### ISSUE TITLE
29399432: xcodebuild - App accessibility isn't loaded

### ISSUE THREAD

**Issue by openradar-mirror** [lionheart/openradar-mirror#16412]
#### Description

When executing UI Automation tests, by using xcodebuild command, around test 170 - 180 executions fails due an error:

>     t =     6.05s             Wait for app to idle
>     t =   126.71s                 App event loop idle notification not received, will attempt to continue.
>     t =   126.71s                 App animations complete notification not received, will attempt to continue.

All next test cases will not execute and all will have this error message:

>     t =     4.43s             Waiting for accessibility to load
>     t =    64.65s                 Assertion Failure: testUITests.swift:21: UI Testing Failure - App accessibility isn't loaded

This issue also exists if the tests were started from Xcode. Issue may be similar to 29393034, where (probably) some resources are not released after tests.

Steps to Reproduce:
1. Using terminal go to folder where the attached project was extracted
2. Execute UI tests using:
xcodebuild -project test.xcodeproj -scheme test -destination "platform=iOS Simulator,name=iPhone 6s,OS=10.1" test
3. Tests should fail after `testExample70`

or:

1. Open attached project using Xcode.
2. Run tests on scheme test.
3. Tests should fail after `testExample70`

Expected Results:
All test should be executed and finished

Actual Results:
After reaching test (around) 170 all next tests are failing due assertion "App accessibility isn't loaded"

Version:
masOS Sierra 10.12.1 (16B2555)
Xcode 8.1 (8B62)

Notes:
I've attached logs from xcodebuild command, Xcode and compressed project with tests.

-
Product Version: Version 8.1 (8B62)
Created: 2016-11-28T11:46:16.634630
Originated: 2016-11-28T00:00:00
Open Radar Link: http://www.openradar.me/29399432

**Comment by openradar-mirror** [comment]
Resolved: No
Modified: 2017-02-21T08:46:33.494330

**Comment by openradar-mirror** [comment]
Resolved: No
Modified: 2017-02-21T08:46:33.494330
